packages/paste-website/src/components/Heading.tsx

```diff
@@ -94,14 +96,40 @@ const StyledAnchorHyperlink = styled.a`
   }
 `;
 
+const StyledHeaderTag = styled.span`
+  margin-left: ${themeGet('space.space40')};
+  padding: ${themeGet('space.space10')} ${themeGet('space.space20')};
+  font-size: ${themeGet('fontSizes.fontSize20')};
+  font-weight: ${themeGet('fontWeights.fontWeightNormal')};
+  text-transform: uppercase;
+  background-color: ${themeGet('colors.colorGray30')};
+  border-radius: ${themeGet('radii.borderRadius10')};
+`;
+
 const AnchoredHeading: React.FC<HeadingProps> = props => {
   // Only generate slugs for headings where children is 'string'
   if (typeof props.children === 'string') {
     const id = slugify(props.children);
+
+    // get tag prop and add flexStyles if tag prop
+    let flexStyles;
+    let {tag} = props;
+    if (tag) {
+      flexStyles = `display: flex; align-items: center;`;
```

Comment:
Could these styles just always be set on the Header component?  Without testing, they seem safe.